src/requirements.txt

```diff
@@ -4,5 +4,6 @@ websockets>=7.0
 pyserial>=3.4
 PySGP30
 PyMLX90614
+Adafruit_AMG88xx
```

Comment:
I was getting `ModuleNotFoundError: No module named 'Adafruit_AMG88xx'` and `ModuleNotFoundError: No module named 'Adafruit_bitfield'` on this branch after re-running `pip install` on this `requirements.txt` file. I had to use the old AMG88xx library:
`sudo python3 -m pip install -e git+https://github.com/adafruit/Adafruit_AMG88xx_python#egg=Adafruit_AMG88xx`

And this additional library:
`sudo python3 -m pip install -e git+https://github.com/adafruit/Adafruit_bitfield_python#egg=Adafruit_bitfield`

Reply:
Interesting, a second attempt with `Adafruit_AMG88xx` worked...

Reply:
Works with ``requirements.txt` too now. Going to hide this conversation as if it never happened.

Reply:
What a fascinating roller coaster of events. Do you know why it happened?

Reply:
Nope, not a clue. As far as I can remember, I uninstalled the old Adafruit library before testing again. If I didn't, or it was unsuccessful, that could be why.

Reply:
New environment, encountered this issue again. Installed requirements.txt on this branch (including Adafruit_AMG88xx), service won't start:

```pytb
Traceback (most recent call last):
  File "src/main.py", line 89, in <module>
    manager.run()
  File "src/main.py", line 40, in run
    self.sensor_process = SensorStream(1, self.sensor_pipe, self.config_file)
  File "/opt/sights/SIGHTSRobot/src/sensor_stream.py", line 53, in __init__
    plugin = importlib.import_module(plugin_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/sights/SIGHTSRobot/src/sensors/amg8833_wrapper.py", line 2, in <module>
    import Adafruit_AMG88xx.Adafruit_AMG88xx as AMG88xx
ModuleNotFoundError: No module named 'Adafruit_AMG88xx'
```

After installing the old libraries mentioned in the original comment, the service was able to start.

`sudo python3 -m pip install -e git+https://github.com/adafruit/Adafruit_AMG88xx_python#egg=Adafruit_AMG88xx`
`sudo python3 -m pip install -e git+https://github.com/adafruit/Adafruit_bitfield_python#egg=Adafruit_bitfield`

Reply:
Resolved by #40 for new installations (or existing ones after running `pip install` against `requirements.txt` again) should you experience the same issue.